src/warplib/implementations/maths/mul.ts

```diff
@@ -118,17 +118,28 @@ export function mul_signed(): void {
           `func warp_mul_signed256{range_check_ptr, bitwise_ptr : BitwiseBuiltin*}(`,
           `    lhs : Uint256, rhs : Uint256) -> (result : Uint256):`,
           `    alloc_locals`,
+          `    # 1 => lhs >= 0, 0 => lhs < 0`,
           `    let (lhs_nn) = uint256_signed_nn(lhs)`,
+          `    # 1 => rhs >= 0, 0 => rhs < 0`,
           `    let (local rhs_nn) = uint256_signed_nn(rhs)`,
```

Comment:
strange local here for rhs_nn but not lhs_nn

Reply:
Yeah, probably not strictly necessary anymore since the cairo compiler got upgraded, but it used to prevent rhs_nn getting revoked